### ISSUE TITLE
Error running database seeder

### ISSUE THREAD

**Issue by connor11528** [reinink/laracon2018#4]
I cloned the project and am getting an error when running the database seeder:

```
php artisan db:seed

In Connection.php line 664:
                                                                               
  SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '2  
  011-03-13 02:27:30' for column 'created_at' at row 1 (SQL: insert into `int  
  eractions` (`type`, `created_at`, `updated_at`, `customer_id`) values (Brea  
  kfast, 2011-03-13 02:27:30, 2011-03-13 02:27:30, 37))                        
                                                                               

In Connection.php line 458:
                                                                               
  SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '2  
  011-03-13 02:27:30' for column 'created_at' at row 1
```

Is there something that I need to change in order to make the date comparisons work?

Thank you for the talk and project code

**Comment by reinink** [comment]
That's really weird. `2011-03-13 02:27:30` is a totally valid date, and really that just gets generated by Laravel anyway via the model timestamps. Sorry, I've got nothing. I'd start Googling. 😞

**Comment by connor11528** [comment]
Okay for sure. I think it might have something to do with the Mysql version on my home machine. I got past it using mysql version 5.7 but then ended up with

```
SQLSTATE[42000]: Syntax error or access violation: 1305 FUNCTION laracon2018.to_char does not exist (SQL: select count(*) as aggregate from `customers` where `birth_date` is not null and to_char(birth_date, 'MMDD') in (0409, 0410, 0411, 0412, 0413, 0414, 0415) and `sales_rep_id` = 1)
```

Going to leave this open in case anyone else experiences this or has other insights, if you don't mind!

**Comment by amadeann** [comment]
This is caused by a little quirk in MySQL's timezones interpretation. Datetime values are not interpreted directly as UTC, but first assumed to be the timezone determined by your system, and then converted to UTC. I kept getting that error as well, but only for March dates in different years - and they always coincided with daylight saving, at early morning hours (quite strangely it didn't matter if it was North American, or European dayligt saving). 

More on the issue here: https://stackoverflow.com/a/25855998/4171578 (including the comments).

I was able to fix it by going to MySQL and executing this command:

    SET GLOBAL time_zone='+0:00'
